test/api_key_test.go

```diff
@@ -109,6 +109,9 @@ func (s *CLITestSuite) TestAPIKey() {
 		{args: "api-key create --description json-output --resource lkc-other1 -o json", fixture: "api-key/45.golden"},
 		{args: "api-key create --description yaml-output --resource lkc-other1 -o yaml", fixture: "api-key/46.golden"},
 
+		// create api-key and use for the resource
+		{args: "api-key create --description my-cool-app --resource lkc-cool1 --use", fixture: "api-key/60.golden"}, // MYKEY4
```

Comment:
Let's run `api-key describe` immediately after to prove that `--use` worked?

Reply:
We might also want to split up big functions like `TestAPIKey()`... it takes way longer to debug when there are so many lines.